docs/edgeql/ddl/attributes.rst

```diff
@@ -65,13 +67,13 @@ Remove a :ref:`schema attribute <ref_datamodel_attributes>`.
 .. eql:synopsis::
 
     [ WITH <with-item> [, ...] ]
-    DROP ATTRIBUTE <name> ;
+    DROP ABSTRACT ATTRIBUTE <name> ;
 
 Description
 -----------
 
-``DROP ATTRIBUTE`` removes an existing schema attribute from the database
-schema.
+``DROP ABSTRACT ATTRIBUTE`` removes an existing schema attribute from
+the database schema.
```

Comment:
Should probably mention that it can drop `INHERITABLE` attributes too, without the need to use the qualifier.